### ISSUE TITLE
Crouzon nose

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9607]
Short, stubby, said to resemble parrot beak

https://omim.org/entry/123500?search=123500&highlight=123500

https://pubmed.ncbi.nlm.nih.gov/35235708/  8/8

**Comment by pnrobinson** [comment]
@azankl
Is there an acceptable way of describing this nose form without using the phrase "parrot-like"?

**Comment by azankl** [comment]
@pnrobinson how about 'beaked nose'?

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001064
- label: Beaked nose
- parents: [Abnormal nasal morphology (HP:0005105)](https://hpo.jax.org/app/browse/term/HP:0005105)
- pmid: PMID:22605842|PMID:35235708
- def: An abnormal form of the nose characterized by a prominent bridge and a downward curve at the tip, said to resemble the beak of a bird.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9607
- synonyms: Parrot-beaked nose(NONE)|Crouzon nose(NONE)

**Comment by pnrobinson** [comment]
I think we should used Convex nasal ridge, which has "beaked" as a synonym.

**Comment by azankl** [comment]
@pnrobinson Yes, that sounds good!
